src/components/InputTodo.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/button-has-type */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/state-in-constructor */
```

Comment:
Kindly suggest not changing linter config. Please check out general suggestions.  ⬆️